Jerrod/Assignments/names.rb

```diff
@@ -0,0 +1,11 @@
+a = {:first_name => "Michael", :last_name => "Choi"}
+b = {:first_name => "John", :last_name => "Supsupin"}
+c = {:first_name => "KB", :last_name => "Tonel"}
+d = {:first_name => "Jessie", :last_name => "De Leon"}
+e = {:first_name => "Jaybee", :last_name => "Balog"}
+names = [a, b, c, d, e]
+
+puts "You got 5 names in the 'names' array"
```

Comment:
What if there were more than 5 names in the array?  Line 8 wouldn't work.  Instead you could try...
```ruby
puts "You got #{names.count} in the 'names' array"
```